math/stringGlues.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <string>
+#include <vector>
+using namspace std;
+
+string stringGlues(vector<string> v) {
+	string full=""; 
+	
+	for(int i=0; i < v.length() - 1; i++)
+	{
+		full = full+v[i];
+		full= full+"glue";
+	}
+	
+	return full + v.pop(); 
+}
+
+int main() {
```

Comment:
Move your StringGlues function into FunMath.h (declaration) and FunMath.cpp (implementation) and remove the file stringGlues.cpp.